src/js/edu-benefits/1990/utils/validations.js

```diff
@@ -274,6 +274,5 @@ export {
   isValidTourOfDuty,
   isValidEmploymentPeriod,
   isValidRotcScholarshipAmount,
-  isValidEducationPeriod,
-  isValidAddressField
```

Comment:
@crwallace - it's a duplicate member. npm lint from the Jenkins docker-based tests caught the issue, but I haven't compared versions or config to determine why the version on Travis didn't. 

Removed from this and will submit in a new PR with better research.